### ISSUE TITLE
About loss

### ISSUE THREAD

**Issue by ph0316** [wyf0912/LLFlow#29]
Hi, I have a question, how do I use L1 loss and maximum likelihood loss at the same time? When I used both together, I encountered the following error, and the resulting image would also be black. I am looking forward to your reply. Thank you!
![image](https://user-images.githubusercontent.com/56188513/188469984-87c09e05-c246-4a2b-8834-2c50b4db8e65.png)

**Comment by wyf0912** [comment]
Hi, you can try to train the model using NLL loss for a few epochs first.
Then train the model using L1 and NLL together. 

You may need to decrease the learning rate since introducing an extra loss.
